checker/manual/units-checker.tex

```diff
@@ -241,8 +252,21 @@ \section{Running the Units Checker\label{units-running}}
 
 \begin{alltt}
   javac -processor org.checkerframework.checker.units.UnitsChecker \ttbs
-        \textit{-Aunits=myproject.qual.MyUnit,myproject.qual.MyOtherUnit} MyFile.java ...
+        \textit{-Aunits=qual.MyUnit,qual.MyOtherUnit} MyFile.java ...
+\end{alltt}
+
+\item
+You can also provide a set of folders that contain units qualifiers using the
+\code{-AunitsDirs} option, and also adding the folders to the bootclass path, for example:
```

Comment:
"adding" => "add".
(I presume you will find all the duplicated text and update it consistently when I make a suggestion in one place.  Thanks!)